docs/cristin.ttl

```diff
@@ -514,14 +542,98 @@
     :mapstonvafield "entityDescription.contributor.role. only the first role of a person is considered." .
 
 :rollekode
-    a            rdf:Property ;
-    rdfs:comment """
+    a               :cristinfield ;
+    rdfs:comment    """
         The role of the respective Contributor for the respective publication affiliated for this role with the respective institution.
         This entry is immutable and does not change in time.
         """ ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :VARBEID_PERSON_STED_ROLLE_Class ;
+    :isignored      "false" ;
+    :mapstonvafield " maps to NVA Role" .
+
+:eierkode
+    a            :cristinfield ;
     rdfs:range   xsd:string ;
-    rdfs:domain  :VARBEID_PERSON_STED_ROLLE_Class ;
-    :isignored   "false" .
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """The institution the currently owns the NVI result. Example UIO.
+    We do not know whether the values of this field are standardized and whether there is a defined set of values for this property's range""" ;
+    :isignored   "true" .
+
+
+:status_kontrollert
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """
+     It indicates if the respective post has been controlled or not. It receives values "J", "N".
+     It is unknown whether it refers to the Cristin entry or the NVI entry.
+    """ ;
+    :isignored   "true" .
+
+:brukernavn_kontrollert
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """
+     The username who controlled the respective entry. Unknown whether the field refers to the NVI entry or the Cristin entry.
+     Unknown whether the user belongs to NSD or to the institution that registered the entry (e.g. NTNU)
+
+    """ ;
+    :isignored   "true" .
+
+:dato_kontrollert
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """
+     The date the respective entry was controlled. Unknown whether the field refers to the NVI entry or the Cristin entry.
```

Comment:
Same. Domain is the NVI processing record for one institution, date refers to when a superuser of said institution did their review of the Cristin entry / result / publication.